### PR TITLE
Feat/notifications app display notifications

### PR DESCRIPTION
**Pull request by themonster2015**
## Description

* Creation of the components for displaying notifications and proper routing added.
* Added settings menu functionality.

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      <Card
+        direction="row"
+        elevation={'1'}
+        radius={16}
+        padding={{ x: 0, y: 8 }}
+        customStyle="pb-16"
+      >
+        <div className={tw('py-4 relative w-full')}>
+          <Text variant="h5" align="center">
+            Notifications
```
Comment (quininez) [inline ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx]: add also the translation utility useTranslation to labels, you can check out other apps to see how, import import { useTranslation } from 'react-i18next';
const { t } = useTranslation('app-notifications');
then in the html <text>{t('Notifications')}</text>

File: ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+}
+
+const NotificationsPage: React.FC<RootComponentProps> = props => {
+  const [searchParams] = useSearchParams();
+  const [message, setMessage] = React.useState('');
+  const [messageType, setMessageType] = React.useState('');
+  const [showMenu, setShowMenu] = React.useState(false);
+
+  React.useEffect(() => {
+    /* uncomment this part when backend data becomes available */
```
Comment (quininez) [inline ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx]: for areas that we need to get back later to modify please leave a comment with @TODO so we can easily find them

File: ui/design-system-core/src/components/BasicInfoCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+interface IBasicInfoCardProps {
+  titleLabel: string;
+  subtitleLabel?: string;
+}
+
+const BasicInfoCard: React.FC<IBasicInfoCardProps> = ({ titleLabel, subtitleLabel }) => {
+  return (
+    <Card padding={8} customStyle="border-none">
+      <div className={tw('flex(& col) justify-center align-center mb-32')}>
+        <BasicCardBox style="bg-grey8 w-[180px] h-[180px] m-auto my-4" round="rounded-xl" />
```
Comment (quininez) [inline ui/design-system-core/src/components/BasicInfoCard/index.tsx]: this card uses the same background on light and dark theme?

**Review by quininez — CHANGES_REQUESTED** [review]

File: ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx (hunk 1 above)
Comment (themonster2015) [reply ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx]: I have added it.

File: ui/design-system-core/src/components/BasicInfoCard/index.tsx (hunk 3 above)
Comment (themonster2015) [reply ui/design-system-core/src/components/BasicInfoCard/index.tsx]: fixed :))

**Review by quininez — APPROVED** [review]

**Pull request merged**